src/debug-panel/demo.html

```diff
@@ -15,28 +15,42 @@
 
     <script type="steal-module">
       import DebugPanel from "~/src/debug-panel/"
+      import Interview from "~/src/models/interview"
+      import TraceMessage from "~/src/models/trace-message"
+      import canViewModel from "can-view-model"
+      import stache from 'can-stache'
+
       import "~/styles.less"
 
-      const interview = {
-        variablesList: [
-          {value: null, name: "Client first name TE", repeating: null},
-          {value: null, name: "Client last name TE", repeating: null},
-          {value: null, name: "Client middle name TE", repeating: null},
-          {value: null, name: "sansAnswer", repeating: null},
-          {value: null, name: "User Avatar", repeating: null},
-          {value: null, name: "User Gender", repeating: null}
+      const render = (data) => {
+        const tpl = stache('<debug-panel interview:from="interview" traceMessage:from="traceMessage" />')
+        document.querySelector('#demo-area').appendChild(tpl(data))
+        return canViewModel('debug-panel')
+      }
+
+      const interview = new Interview({
+        vars: [
+          {values: ["Mardy"], name: "Client first name TE", repeating: null},
```

Comment:
answers are indexed by 1 by default, the values array starts as `[null]`, so those should be `[null, "Mardy"]`